tests/files.ts

```diff
@@ -50,6 +53,8 @@ test.serial(
 	(t) => {
 		const { body } = t.context;
 		t.is(body.length, 1, "Should have one file");
+		unstableValues.add(body.id);
```

Comment:
This will cause an error. If `body` is an array, `body.id` will be undefined. Additionally, there's no need to add the ID again since it's already been on line 38.